ibm/mas_devops/roles/sls_install/tasks/main.yml

```diff
@@ -161,12 +162,24 @@
 # -----------------------------------------------------------------------------
 - include_tasks: "tasks/sls-verify.yml"
 
+# Adding some delay to get SLS LicenseService ready
```

Comment:
Do not add any fixed delays like this.  The code works today without any, there is no reason to add a fixed 5 minute delay,we already have code that waits until SLS is ready by polling the state of the CR.  

The role is idempotent, if I run the role after successfully installing it should execute and complete super fast because SLS is already installed.  With this change we just added 5 minutes even when the role needs to do nothing.

Reply:
Sure. Reverting back the change.